src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -0,0 +1,645 @@
+# Dynamic proxy configuration
+
+As part of the application deployment to ConductR, some of these application may provide one or more endpoints to be exposed via proxy.
+
+ConductR uses [HAProxy](http://www.haproxy.org/) as its proxying solution.
+
+The ConductR HAProxy bundle is used to maintain the cluster-wide information (e.g. what bundle is running at which server among many other things) so the correct proxy configuration can be generated.
+
+Whenever there's a need to update the proxy configuration due to cluster-wide information change (e.g. new application being deployed and started), the ConductR HAProxy bundle will ensure HAProxy is configured with the updated information. In other words, the configuration of the HAProxy will be dynamically updated to reflect the correct information within the ConductR cluster.
+
+The operator will also have the option to customise the HAProxy configuration based on the information supplied by the ConductR HAProxy bundle.
```

Comment:
I think it'd be good to start this doc with a sample command showing how easier it is to configure conductr-haproxy with new configuration... we need to get people excited about this feature. :-)